portfolio/src/main/webapp/script.js

```diff
@@ -26,21 +26,25 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-//add Hello Nzinga! to the page
-//async function getHelloNzingaUsingAsyncAwait() {
-//  const response = await fetch('/data');
-//  const quote = await response.text();
-//  document.getElementById('HelloNzinga-container').innerText = quote;
-//}
-
 function getHelloNzingaUsingAsyncAwait() {
-    fetch('/data')  // sends a request to /my-data-url
-    .then(response => response.json()) // parses the response as JSON
+    fetch("data") // sends a request to /my-data-url
```

Comment:
I'm guessing the slash at the beginning not required?

Reply:
yes it works with out it